### PR TITLE
feat(schemas,console): add mfa interaction log key

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add the log key that will be used in MFA feature. This helps to reduce the size of https://github.com/logto-io/logto/pull/4551

## Testing

N/A

## Checklist

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/consts/logs.ts [hunk 1]
```diff
@@ -22,6 +22,12 @@ export const auditLogEventTitle: Record<string, Optional<string>> &
   'Interaction.ForgotPassword.Profile.Update': 'Patch update forgot-password interaction profile',
   'Interaction.ForgotPassword.Submit': 'Submit forgot-password interaction',
   'Interaction.ForgotPassword.Update': 'Update forgot-password interaction',
+  'Interaction.ForgotPassword.BindMfa.TOTP.Create': undefined,
```
Comment (simeng-li) [inline packages/console/src/consts/logs.ts:25]: Let the copilot generate those human-readable names?

**Review by simeng-li — APPROVED** [review]
LGTM

**Review by darcyYe — APPROVED** [review]

File: packages/console/src/consts/logs.ts (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/consts/logs.ts:25]: @xiaoyijun leave it to you.

File: packages/console/src/consts/logs.ts [hunk 2]
```diff
@@ -46,6 +58,12 @@ export const auditLogEventTitle: Record<string, Optional<string>> &
   'Interaction.SignIn.Profile.Update': 'Patch Update sign-in interaction profile',
   'Interaction.SignIn.Submit': 'Submit sign-in interaction',
   'Interaction.SignIn.Update': 'Update sign-in interaction',
+  'Interaction.SignIn.BindMfa.TOTP.Create': undefined,
+  'Interaction.SignIn.BindMfa.TOTP.Submit': undefined,
```
Comment (gao-sun) [inline packages/console/src/consts/logs.ts:62]: should be `Totp`?

**Review by gao-sun — APPROVED** [review]

**Pull request merged**